### PR TITLE
Move the Gitter badge to the header

### PR DESCRIPTION
**Pull request by joelwallis**
Just because it's cooler and looks more organized 😄

Preview here: https://github.com/joelwallis/exercism-xecmascript/tree/move-gitter-badge-to-header

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# xECMAScript [![Build Status](https://travis-ci.org/exercism/xecmascript.svg?branch=master)](https://travis-ci.org/exercism/xecmascript)
-
-[![Join the chat at https://gitter.im/exercism/xecmascript](https://badges.gitter.im/exercism/xecmascript.svg)](https://gitter.im/exercism/xecmascript?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+# xECMAScript [![Build Status](https://travis-ci.org/exercism/xecmascript.svg?branch=master)](https://travis-ci.org/exercism/xecmascript) [![Join the chat at https://gitter.im/exercism/xecmascript](https://badges.gitter.im/exercism/xecmascript.svg)](https://gitter.im/exercism/xecmascript?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (tejasbubane) [inline README.md]: Can you also put a newline between the title `xECMAScript` and the `build-status` badge? Will look like https://github.com/exercism/exercism.io/

Comment (joelwallis) [inline README.md]: That's a really good idea! I'll definitely do it!

**Comment by joelwallis** [comment]
How does it look now, @tejasbubane?

**Comment by joelwallis** [comment]
Preview here: https://github.com/joelwallis/exercism-xecmascript/tree/move-gitter-badge-to-header

**Comment by matthewmorgan** [comment]
Fantastic.  Thanks @joelwallis, and @tejasbubane for the feedback.  💯

**Pull request merged**